content/applications/inventory_and_mrp/plm/manage_changes/eco_type.rst

```diff
@@ -39,105 +49,86 @@ the following information:
 
-Then, add an approver in the :guilabel:`Approvers` section, by clicking :guilabel:`Add a line`, and
-specifying the :guilabel:`Role` of the reviewer, their :guilabel:`User`, and :guilabel:`Approval
-Type`.
-
-Make sure at least one approver is configured with the :guilabel:`Approval Type`: :guilabel:`Is
-required to approve`.
+Next, add an approver in the :guilabel:`Approvers` section, by clicking :guilabel:`Add a line`, and
+specifying the :guilabel:`Role` of the reviewer, the :guilabel:`User`, and an :guilabel:`Approval
+Type` of :guilabel:`Is required to approve`. See :ref:`more about approval types.
+<plm/approvals/approval-type>`
 
 The approver listed is automatically notified when |ECOs| are dropped in the stage specified in the
 pop-up window. Once finished, click :guilabel:`Save & Close`.
 
-.. example::
-   In the |ECO| type `New Product Introduction`, the verification stage `Validated` is configured by
-   clicking the :guilabel:`⚙️ (gear)` icon, and selecting :guilabel:`Edit`. Doing so opens the
-   :guilabel:`Edit: Validated` pop-up window.
+.. seealso::
+   :doc:`../management/approvals`
 
-   By adding the `Engineering manager` as an approver, only |ECOs| approved by this user can proceed
-   to the next stage, and have the changes applied on the production |BOM|.
+.. _plm/eco/closing-stage:
 
-   Additionally, check the :guilabel:`Allow to apply changes` option to ensure proper behavior.
+Closing stages
+--------------
 
-   .. image:: eco_type/verification-stage.png
-      :align: center
-      :alt: Show "Allow to apply changes" option is checked.
+Click an |ECO| type from :menuselection:`PLM app --> Overview` to open the Kanban view of |ECOs| of
+this type.
 
-Closing stage
--------------
-
-Configure a closing stage by opening the :guilabel:`Edit: [stage]` pop-up window. To do so, hover
-over the intended stage and click the :guilabel:`⚙️ (gear)` icon that appears in the top-right
-corner. Then, click :guilabel:`Edit` from the drop-down menu.
-
-On the :guilabel:`Edit: [stage]` pop-up window, select the check boxes for :guilabel:`Folded in
-kanban view`, :guilabel:`Allow to apply changes` and :guilabel:`Final Stage`.
+To configure a closing stage that applies |BOM|, edit the stage and tick the boxes for
+:guilabel:`Folded in kanban view`, :guilabel:`Allow to apply changes`, and :guilabel:`Final Stage`.
+When ECO cards are placed in a stage that will *Allow to apply changes*, then any |BOM| and
+operations changes validated in the |ECOs| will immediately be put into effect. To create a
+cancelled stage, create or edit a stage and tick the boxes for :guilabel:`Folded in kanban view` and
+:guilabel:`Final Stage`. |ECOs| in this stage are removed from the pipeline, but will not make
+changes.
 
 .. example::
    The closing stage, `Effective` is configured by checking the :guilabel:`Folded in kanban view`,
    :guilabel:`Allow to apply changes`, and :guilabel:`Final Stage` options
 
 .. image:: eco_type/closing-stage.png
```

Comment:
Optional! It seems like it would be relevant to include the image in this example admonition :)
```suggestion
.. example::
   The closing stage, `Effective` is configured by checking the :guilabel:`Folded in kanban view`,
   :guilabel:`Allow to apply changes`, and :guilabel:`Final Stage` options

   .. image:: eco_type/closing-stage.png
```